service.exists with systemd fails

On a systemd with systemd using SystemdService class, `service.exists` property fails with

````
 AssertionError: Unexpected exit code 2 for CommandResult(backend=<testinfra.backend.docker.DockerBackend object at 0x7f23a47a8520>, exit_status=2, command=b'systemctl list-unit-files | grep -q"^avahi-daemon"', _stdout=b'', _stderr=b"grep: invalid option -- '^'\nUsage: grep [OPTION]... PATTERNS [FILE]...\nTry 'grep --help' for more information.\n")
assert 2 in [0, 1]
+  where 2 = CommandResult(backend=<testinfra.backend.docker.DockerBackend object at 0x7f23a47a8520>, exit_status=2, command=b'syst...=b"grep: invalid option -- '^'\nUsage: grep [OPTION]... PATTERNS [FILE]...\nTry 'grep --help' for more information.\n").rc
````

--- a/testinfra/modules/service.py
+++ b/testinfra/modules/service.py
@@ -176,7 +176,7 @@ class SystemdService(SysvService):
 
     @property
     def exists(self):
-        cmd = self.run_test('systemctl list-unit-files | grep -q"^%s"', self.name)
+        cmd = self.run_test('systemctl list-unit-files | grep -q "^%s"', self.name)
         return cmd.rc == 0
 
     @property

